README.md

```diff
@@ -31,4 +31,73 @@ Po zamknięciu zapisów moderatorzy roku przypisują studentów do grup uwzględ
 - Może przydzielać role
 - Może dodawać, usuwać i edytować konta studentów
 
+
+# Typy encji
+
+## Student
+
+Reprezentuje aktora Student.
+- Imię
+- Nazwisko
+- E-mail
+- Numer indeksu
+
+## Moderator
+
+Podklasa typu encji _Student_
+
+Reprezentuje aktora Moderator.
+
+## Administrator
+
+Podklasa typu encji _Moderator_
+
+Reprezentuje aktora Administrator.
```

Comment:
Zastanawiam się czy tworzenie encji bez własności ma sens (moderator, administrator). Chciałbym też odseparować dane logowania od użytkownika, co pozwoli nam np stworzyć logowanie przez OAuth obok maila jeżeli będziemy chcieli.

Co myślisz o takim podziale?

## Dane logowania

- Mail
- Hash hasła

## Użytkownik

- Imię
- Nazwisko
- Nr indeksu

## Rola

- Nazwa

Reply:
Rzeczywiście robienie takiego zagnieżdżenia ma sens jedynie wtedy, gdy różne podklasy mają różne relacje, a chyba tak tu nie będzie.

Reply:
Co do danych logowania, czy dajemy jakiś salt lub coś takiego? Nie wiem jak to dokładnie wygląda tutaj.

Reply:
Pewnie weźmiemy jakieś gotowe rozwiązanie, dodanie saltu zazwyczaj jest standardem w libkach.

Reply:
Ok then.